superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1252,6 +1430,24 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                 />
                 {renderPreferredSelector()}
                 {renderAvailableSelector()}
+                <Upload
+                  name="databaseFile"
+                  id="databaseFile"
+                  data-test="database-file-input"
+                  accept=".yaml,.json,.yml,.zip"
+                  // upload is handled by hook
```

Comment:
remove this comment

Reply:
Good catch, removed in [`this commit`](https://github.com/apache/superset/pull/19314/commits/a5d4761a49da4c4799022ac437698431d5b267f3).